shark-graph/src/test/java/shark/LongScatterSetTest.kt

```diff
@@ -0,0 +1,131 @@
+package shark
+
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Ignore
+import org.junit.Test
+import shark.internal.hppc.HHPC
+import shark.internal.hppc.LongScatterSet
+
+class LongScatterSetTest {
+
+  @Test fun `verify add operation`() {
+    val set = LongScatterSet()
+    verifyAddOperation(set)
```

Comment:
This test doesn't express what its testing, ie "verify add operation" doesn't tell me anything. Same for verifyRemoveOperation

Reply:
@Armaxis same as before, can you provide a better name?